File: src/org/genyris/core/PairEquals.java
package org.genyris.core;


public class PairEquals extends Pair {

    public PairEquals(Exp car, Exp cdr) {
        super(car, cdr);
    }
	public Symbol getBuiltinClassSymbol(Internable table) {
		return table.PAIREQUAL();
	}

}

File: src/org/genyris/core/PairEquals.kt
package org.genyris.core


class PairEquals(car: Exp?, cdr: Exp?) : Pair(car, cdr), Comparable<Any?> {
    override fun getBuiltinClassSymbol(table: Internable): Symbol? {
        return table.PAIREQUAL()
    }
}
